# Command to camera servo controller
std_msgs/Header header
float32 pan_angle   # -90 to 90
float32 tilt_angle  # -30 to 90
bool reset_to_center
